packages/drivers/odsp-driver/src/fetchSnapshot.ts

```diff
@@ -287,6 +290,8 @@ async function fetchLatestSnapshotCore(
 		const fetchSnapshotForLoadingGroup = isSnapshotFetchForLoadingGroup(loadingGroupIds);
 		const eventName = fetchSnapshotForLoadingGroup ? "TreesLatestForGroup" : "TreesLatest";
 		const internalFarmType = checkForKnownServerFarmType(odspResolvedUrl.siteUrl);
+		const isRedemptionNonDurable: boolean =
+			odspResolvedUrl.shareLinkInfo?.isRedemptionNonDurable === true;
```

Comment:
put the value in where we are declaring perfevent instead of where we are ending the event because if error occurs, then end will not occur, rather cancel would occur. So we need to put this value before.